guide/private.md

```diff
@@ -0,0 +1,19 @@
+# Private Messages
+
+```
+-brand-name = Firefox
+
+app-title = { -brand-name }
+has-updated = { -brand-name } has been updated.
+```
+
+Private messages are similar to regular messages but they can only be used as
+references in other messages. Their identifiers start with at least one dash
+`-` like in the example above: `-brand-name`. The runtime cannot retrieve
+private messages directly. They are best used to define glossary terms which
```

Comment:
Just dropping the `glossary` here alltogether sounds good enough for me.

Reply:
Just to make sure I understand this comment, you'd rather see `…to define terms which…`?

Reply:
yes.